src/index.ts

```diff
@@ -7,17 +7,18 @@
 export { default as Multipass } from "./multipass/Registrar";
 export { default as InstanceBase } from "./rankify/InstanceBase";
 export { default as InstancePlayer } from "./rankify/Player";
-export { default as MultipassBase, NameQuery } from "./multipass/MultipassBase";
+export { default as MultipassBase, type NameQuery } from "./multipass/MultipassBase";
 export { default as Registrar } from "./multipass/Registrar";
 
 // Utility exports
 export * from "./utils";
 export * from "./types";
 export * from "./rankify/MAODistributor";
+export { default as RankTokenClient } from "./rankify/RankToken";
 
 // Type exports
 export { gameStatusEnum } from "./rankify/InstanceBase";
-export { MAOInstances } from "./types/contracts";
+export { type MAOInstances } from "./types/contracts";
```

Comment:
these type prefixes seem to be redundant, MAOInstances, NameQuery etc are already interfaces, why casting them in type?